base/collect_database.go

```diff
@@ -47,32 +45,14 @@ type Params struct {
 	Value string `json:"value" yaml:"value"`
 }
 
-const startingTimeString = "StartingTime"
-const elapsedTimeString = "ElapsedTime"
-
-// ToDo: Iter8 wiki is a great place to document the following (whichs seems IBM Code Engine specific) ... move it there
+const (
+	StartingTime = "StartingTime"
+	ElapsedTime  = "ElapsedTime"
+	TimeLayout   = "Jan 2, 2006 at 3:04pm (MST)"
```

Comment:
Please change `TimeLayout` to `timeLayout`

Reply:
I can make this change. Is there a reason behind this though?